src/debuggee.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var crypto = require('crypto');
+var path = require('path');
+var pjson = require('../package.json');
+var StatusMessage = require('./status-message.js');
+
+/**
+ * Creates a Debuggee service object.
+ * @ref https://cloud.google.com/debugger/api/reference/rest/v2/Debuggee
+ *
+ * @param {string} projectId - Google Cloud Project ID
+ * @param {string} uid - unique identified for the source code on this instance.
+ * @param {?object} serviceContext
+ * @param {string} serviceContext.service - A string identifying the service/
+ *     module that this instance belongs to.
+ * @param {string} serviceContext.version - A string identifying the version of
+ *     the service.
+ * @param {?object} sourceContext
+ * @param {?string} description - A user specified string identifying this
+ *     debuggable instance.
+ * @param {?string} errorMessage - A error string to register this as a erroring
+ *     debuggable instance. This is useful if we have a problem starting the
+ *     debugger support, and want to report to the API so that the users has a
+ *     way of noticing.
+ */
+function Debuggee(projectId, uid, serviceContext, sourceContext, description,
+                  errorMessage) {
+  if (!(this instanceof Debuggee)) {
+    return new Debuggee(projectId, uid, serviceContext, sourceContext,
+                        description, errorMessage);
+  }
+
+  if (typeof projectId !== 'string') {
```

Comment:
Suggestion: since lodash is already included as a dependency use `isString` for brevity of expression